docs/tutorials/tabular_prediction/tabular-indepth.md

```diff
@@ -165,6 +165,12 @@ predictor.leaderboard(extra_info=True, silent=True)
 
 The expanded leaderboard shows properties like how many features are used by each model (`num_features`), which other models are ancestors whose predictions are required inputs for each model (`ancestors`), and how much memory each model and all its ancestors would occupy if simultaneously persisted (`memory_size_w_ancestors`). See the [leaderboard documentation](../../api/autogluon.predictor.html#autogluon.tabular.TabularPredictor.leaderboard) for full details.
 
+To show scores for other metrics, you can specify the `extra_metrics` argument when passing in `test_data`:
```

Comment:
maybe comment on the negative metric issue here as well.

Reply:
also could be helpful to clarify score_test/val are the `eval_metric` originally specified (or automatically chosen by AutoGluon) in `fit()`